assets/css/variables.scss

```diff
@@ -67,3 +67,4 @@ $button-secondary-outline-active-bg: #d9d9d93d;
 $text-button-blue-hover-color: #c1e4ff14;
 $text-button-grey-hover-color: #f8f9fa14;
 $alert-yellow: #ffdd00;
+$accessibility: #165c96;
```

Comment:
* Should this be `accessibility-blue` to align with the naming of things like `alert-yellow`? Since color variables share a global namespace with non-color variables, I think it helps to have color names in them.
* It seems odd that this is called "accessibility" when its only current use has nothing to do with accessibility. Though if that's the official color name in our designs I guess we can go with it.

Reply:
I agree it is odd but I was aligning with the [designs](https://www.figma.com/design/P6RZ1AJWk7P98D6IFDK9Ik/PA%2FESS-Modernization?node-id=9551-12261&t=OnjSko2XRQ2l3oMP-4) like you mentioned.